src/cli.ts

```diff
@@ -117,10 +118,63 @@ function debugCliInfo(version: string, channel: string, env: Env, config: IConfi
   );
 }
 
+interface VersionDetailI {
```

Comment:
should this be just `VersionDetail` not `VersionDetailI` with the capital I at the end?